demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -220,6 +221,97 @@ def generate_setup_section(yaml_data: dict):
+
+    for conf in yml_data.get('configuration', []):
+        if conf.get('display') == display:
+            return conf
+    return None
+
+
+def generate_mirroring_section(yaml_data: dict):
+    """
+
+    Args:
+        yaml_data: dict representing the yml file of the integration.
+
+    Returns: markdown section of Incident Mirroring.
+
+    """
+    integration_name = format(yaml_data['display'])
+    directions = {
+        'None': 'Turns off incident mirroring.',
+        'Incoming': 'Any changes in CrowdStrike Falcon incidents (state, status, tactics, techniques, objectives, '
+                    'tags, hosts.hostname) will be reflected in XSOAR incidents.',
+        'Outgoing': 'Any changes in XSOAR incidents will be reflected in CrowdStrike Falcon incidents (tags, status).',
+        'Incoming And Outgoing': 'Changes in XSOAR incidents and CrowdStrike Falcon incidents will be reflected in both directions.'
+    }
+
+    section = [
+        '## Incident Mirroring',
+        '',
+        f'You can enable incident mirroring between Cortex XSOAR incidents and {integration_name} incidents (available from Cortex XSOAR version 6.0.0).',
+        'To setup the mirroring follow these instructions:',
+        '1. Navigate to **Settings** > **Integrations** > **Servers & Services**.',
+        f'2. Search for {integration_name} and select your integration instance.',
+        '3. Enable Fetches incidents.'
+    ]
+
+    index = 4
+    if is_configuration_exists(yaml_data, 'Incidents fetch query'):
+        section.append(
+            f'{index}. Optional: You can go to the Incidents fetch query parameter and select the query to fetch the incidents from {integration_name}.')
+        index = index + 1
+    if is_configuration_exists(yaml_data, 'Mirroring tag'):
+        section.append(f'{index}. Optional: You can go to the Mirroring tag parameter and select the tags used to '
+                       f'mark incident entries to be mirrored.')
+        index = index + 1
+
+    # Mirroring direction
+
+    direction_conf = is_configuration_exists(yaml_data, 'Mirroring Direction')
+    if direction_conf:
+        options = []
+        for option in direction_conf.get('options', []):
+            options.append({'Option': option, 'Description': directions.get(option, '')})
+        dir_text = f'{index}. In the Mirroring Direction integration parameter, select in which direction the incidents should be mirrored:'
+        index = index + 1
+        section.append(dir_text)
+        section.extend(generate_table_section(title='', data=options, horizontal_rule=False, numbered_section=True))
+
+    # Close Mirrored XSOAR Incident param
+
+    if is_configuration_exists(yaml_data, 'Close Mirrored XSOAR Incident'):
+        section.append(
+            f'{index}. Optional: Check the Close Mirrored XSOAR Incident integration parameter to close the Cortex'
+            f' XSOAR incident when the corresponding incident is closed in {integration_name}.')
```

Comment:
```suggestion
            f'{index}. Optional: Check the Close Mirrored XSOAR Incident integration parameter to close the Cortex'
            f' XSOAR incident when the corresponding %%incident type%% is closed in {integration_name}.')
```
The relevant incident type/s should be added here. There can be more than one incident type for the mirroring. This should be checked and added here by their names.